### PR TITLE
Add `.well-known/security.txt` file

### PR DESCRIPTION
**Pull request by carols10cents**
Follow-up work to #8791 now that the page is live!

### CODE REVIEW

**Comment by carols10cents** [comment]
uhhhh the tests are failing because linking with cc failed because there's no space left..... does a cache somewhere need cleared?

**Review by Turbo87 — APPROVED** [review]

File: public/.well-known/security.txt [hunk 1]
```diff
@@ -0,0 +1,2 @@
+Contact: https://crates.io/policies/security
+Expires: 2025-06-25T00:00:00.000Z
```
Comment (LawnGnome) [inline public/.well-known/security.txt:2]: We're 100% going to forget this, aren't we?

I wonder if we should have a scheduled GitHub action that monitors this and tells us if it's close. (I can write that after this gets merged.)

**Review by LawnGnome — APPROVED** [review]

**Comment by LawnGnome** [comment]
> uhhhh the tests are failing because linking with cc failed because there's no space left..... does a cache somewhere need cleared?

(At least) #9030 also failed for the same reason. Weird.

I'll open an issue and see what I can figure out. (Decidedly not a CI or GitHub Actions expert over here.)

File: public/.well-known/security.txt (hunk 1 above)
Comment (Turbo87) [reply public/.well-known/security.txt:2]: hmm, that's a good point. I guess alternatively the file is small enough that we could also generate it on startup and serve it from the axum server 🤔

Comment (carols10cents) [reply public/.well-known/security.txt:2]: well the point is that a person should look at it every once in a while.........

**Review by jtgeibel — APPROVED** [review]

**Pull request merged**